framework/analytics/__init__.py

```diff
@@ -72,6 +75,8 @@ def build_page(rex, kwargs):
         return None
 
 
+@run_postcommit(once_per_request=False, celery=True)
```

Comment:
I think you're right @sloria. I just tried to merge master into my feature branch and it just reverted all my changes and had a bunch of merge conflicts. I'm trying again.